fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -516,16 +512,8 @@ class HomeFragment : Fragment() {
      * doesn't get run right away which means that we won't draw on the first layout pass.
      */
     private fun updateSessionControlView() {
-        if (browsingModeManager.mode == BrowsingMode.Private) {
-            binding.root.consumeFrom(requireContext().components.appStore, viewLifecycleOwner) {
-                sessionControlView?.update(it)
-            }
-        } else {
-            sessionControlView?.update(requireContext().components.appStore.state)
```

Comment:
Do we no longer need this update outside of a `consumeFrom`?

Reply:
Good catch! Looks like this was [indeed intentional](https://github.com/mozilla-mobile/firefox-android/commit/865eedddd3389480abddcde5c7d7200d33826096#diff-fbc1255e297417e1494bc57e3a8776e7e74fb16022729bfee4d76c10cc3e06fd), so I'll leave it for now with a small change to logic for readability. Hopefully this will go away shortly as we continue to Composify